src/spirc.rs

```diff
@@ -172,6 +247,10 @@ impl SpircInternal {
                     self.is_active = false;
                     self.player.stop();
                 }
+
+                if frame.has_ident() && frame.has_goodbye() {
```

Comment:
I think I'll just remove this. I forgot why I even added it now, so it probably doesn't make any sense.

`kMessageTypeGoodbye` occurs when I log out on my android phone, so I'll add that case.